juju/juju
# Consume offer

## Description of change

The following commit bumps to the latest charm.v6 dep, which takes
the offer url from the juju package and puts it into the charm.v6
dep so that we can easily validate offer urls from the bundle data
perspective without having to duplicate code. With this in mind
we also change the juju code to use that new changes.

Drive by fixes include cleaning up whitespace on lines

## QA steps

```console
export JUJU_DEV_FEATURE_FLAGS=bundle-cmr
juju bootstrap lxd test --no-gui

juju deploy mysql
juju status --relations
```
You should see mysql in the relation section
```console
juju offer mysql:db
juju add-model other
juju switch other
juju deploy ./testcharms/charm-repo/bundle/wordpress-with-saas/bundle.yaml
```
